cranelift-codegen/meta/src/isa/riscv/recipes.rs

```diff
@@ -0,0 +1,259 @@
+use std::collections::HashMap;
+
+use crate::cdsl::formats::FormatRegistry;
+use crate::cdsl::instructions::Predicate;
```

Comment:
In x86/recipes.rs, `instructions::Predicate` is imported as `InstructionPredicate`, so it'd be nice to do that here too to keep the different architectures consistent. In fact, would it make sense to just rename this `Predicate` to `InstructionPredicate` at the source so that we don't need to rename it in the imports?

Reply:
Good call, I've hesitated for long with this, but your arguments make sense to me. Will rename this and PredicateNumber as well.